apps/s_client.c

```diff
@@ -3467,4 +3509,25 @@ static int ldap_ExtendedResponse_parse(const char *buf, long rem)
     return ret;
 }
 
+static char *base64encode (const void *buf, size_t len) {
+
+    /* BASE64 encoder - used only for encoding basic proxy authentication credentials
```

Comment:
Put the block comment BEFORE the function.  Use "/*" and "*/" on lines by themselves. This might be a generally useful function to move into the crypto lib.  Perhaps @levitte has suggestions for the name?

Reply:
Requested change made.